### PR TITLE
[plugin.video.liveleak] 0.0.7

### PR DESCRIPTION
**Pull request by Botster**
### Description
This is a fixed, updated, and working version of the original LiveLeak addon from "oneadvent" (https://github.com/oneadvent/plugin.video.liveleak) which is apparently no longer maintained since my pull request there has gone unanswered for two weeks.

It provides an interface to videos on Liveleak.com.

### Checklist:
- [ X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document

*****
CONTRIBUTING.md does not explain how to target the appropriate repo branch (ie. Jarvis, Krypton, etc.) I have found the process of learning how to submit an addon _very_ confusing. I still don't know and have probably done this incorrectly.
*****

- [X ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.liveleak/addon.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.liveleak"
+    name="LiveLeak"
+    version="0.0.7"
+    provider-name="Botster">
+    <requires>
+        <import addon="xbmc.python" version="2.6.0"/>
```
Comment (enen92) [inline plugin.video.liveleak/addon.xml]: xbmc.python should be version `2.24.0` to match Jarvis

File: plugin.video.liveleak/addon.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Watch LiveLeak.com Videos</summary>
```
Comment (enen92) [inline plugin.video.liveleak/addon.xml]: default now is `en_GB`

File: plugin.video.liveleak/default.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+import urllib, urllib2, urlparse, re
+from HTMLParser import HTMLParser
+
+# Kodi libraries
+import xbmc, xbmcplugin, xbmcgui, xbmcaddon
+
+# Identifiers
+BASE_URL = sys.argv[0]
+ADDON_HANDLE = int(sys.argv[1])
+addon         = xbmcaddon.Addon('plugin.video.liveleak')
```
Comment (enen92) [inline plugin.video.liveleak/default.py]: You can use just `xbmcaddon.Addon()`

File: plugin.video.liveleak/default.py [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+# --- Helper functions ---
+
+def log(txt):
+    """
+    Write text to Kodi log file.
+    :param txt: text to write
+    :type txt: str
+    """
+    message = '%s: %s' % (ADDON_NAME, txt.encode('ascii', 'ignore'))
+    xbmc.log(msg=message, level=xbmc.LOGNOTICE)
```
Comment (enen92) [inline plugin.video.liveleak/default.py]: default log level should be set to `xbmc.LOGDEBUG`

**Comment by enen92** [comment]
Looks okay to go. Please adjust my comments to merge.
Thank you for keeping the plugin alive.

File: plugin.video.liveleak/addon.xml (hunk 1 above)
Comment (Botster) [inline plugin.video.liveleak/addon.xml]: Corrected. But just FYI, http://kodi.wiki/view/Python_Development says:

> 8.1 Version
> Kodi comes with Python 2, version 2.6 or higher depending on platform. 

Thank you.

File: plugin.video.liveleak/addon.xml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    <requires>
+        <import addon="xbmc.python" version="2.24.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Watch LiveLeak.com Videos</summary>
+        <description lang="en_GB">Redefining The Media</description>
+        <language>en_GB</language>
```
Comment (enen92) [inline plugin.video.liveleak/addon.xml]: Language should be en. The en_GB stuff is only for localizedstrings/translations(so, only for summary, description and translations if you had them). Language is for the video content and should be en. Sorry if my previous comment was confusing.

File: plugin.video.liveleak/addon.xml (hunk 1 above)
Comment (enen92) [inline plugin.video.liveleak/addon.xml]: Sure but that refers to the internal python API version, which is different for every release. See here: http://kodi.wiki/view/addon.xml

File: plugin.video.liveleak/addon.xml (hunk 5 above)
Comment (Botster) [inline plugin.video.liveleak/addon.xml]: No problem. Maybe I got it now.

**Pull request merged**